### PR TITLE
feat: Edukaan login

### PR DESCRIPTION
**Pull request by Remon-Refaat**
- Add the Environment as a variable in the command line, the default will be Stage. We can run the script as:
mvn clean test -Pedukaan -Denvironment=local **-DtestingEnvironment=stage** 
                          -DappPath=/Users/remonrefaat/Desktop/Tradeling/Project
- Creating the registration flow using APIs.
- Login Scenario.

### CODE REVIEW

File: src/main/resources/edukaanApp/apiRequestBody/uploadTradeLicence.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "certificates": [
+{
+  "attachments": [
+{
+  "id": "62552d73129bb6001cf9033b",
+  "name": "1164a61302460f18a9e69bbdff96aa5b.pdf",
+  "url": "https://s3.eu-west-1.amazonaws.com/backend-stage-media-storage-origin/module-catalog-pim/.tmp/2552d0d4cdfeece9dbd6f68/buyer-company-setting/1164a61302460f18a9e69bbdff96aa5b.pdf"
```
Comment (MohmedEmad) [inline src/main/resources/edukaanApp/apiRequestBody/uploadTradeLicence.json:8]: @Remon-Refaat we already have same data and method for buyerApp `documentupload.json` & `RegistrationRequests`  should we reuse it? @harshprasad92

**Review by MohmedEmad — COMMENTED** [review]
@harshprasad92 @Remon-Refaat Should we keep all common methods / data / APIs for buyerApp and edukaanApp in one place? for easy maintenance and avoid redundancy?

Or keeping them separate to be more readable?

File: src/main/resources/edukaanApp/apiRequestBody/uploadTradeLicence.json (hunk 1 above)
Comment (Remon-Refaat) [reply src/main/resources/edukaanApp/apiRequestBody/uploadTradeLicence.json:8]: @MohmedEmad They are different. In the mobileApp, the API will take TradeHolder, TradeLicense, and VatCertificate. In Edukaan, the API will take only Tradelicense.

**Comment by Remon-Refaat** [comment]
> common methods / data / APIs

@MohmedEmad We already keep the common methods and APIs in one place, we separate only what is related to each of them.

**Review by MohmedEmad — APPROVED** [review]

**Pull request merged**